roles/confluent.control_center/defaults/main.yml

```diff
@@ -1,13 +1,11 @@
 control_center_open_file_limit: "{{open_file_limit}}"
 
-control_center_ssl_java_arg_buildout: "-Djavax.net.ssl.keyStore={{keystore_path}} -Djavax.net.ssl.trustStore={{truststore_path}} -Djavax.net.ssl.keyStorePassword={{keystore_storepass}} -Djavax.net.ssl.trustStorePassword={{truststore_storepass}}"
```

Comment:
should it refer to control_center_keystore_path etc. here? Somehow this code got promoted to 5.4.0-post and this part caused the c3 install to fail

Reply:
ah yes that should use control_center_keystore_path instead of keystore_path but this line will get removed as part of the PR